packages/hardhat-core/src/builtin-tasks/compile.ts

```diff
@@ -157,6 +158,18 @@ subtask(TASK_COMPILE_SOLIDITY_READ_FILE)
     }
   );
 
+/**
+ * This task is meant to be overriden by potential plugins
+ * It allows to transform an import name
+ */
+subtask(TASK_COMPILE_TRANSFORM_IMPORT_NAME)
+  .addParam("importName", undefined, undefined, types.string)
```

Comment:
Is importName a sourceName? Or is it the literal that's included in the import construct?

Reply:
The string literal in the import statement.